dbt_semantic_interfaces/transformations/proxy_measure.py

```diff
@@ -63,6 +63,7 @@ def transform_model(semantic_manifest: PydanticSemanticManifest) -> PydanticSema
                     )
                     metric.name = measure.name
                     metric.type_params.measure = PydanticMetricInputMeasure(name=measure.name)
+                    metric.type_params.expr = measure.expr or measure.name
```

Comment:
Is this not covered by one of the other transformations that patrick added? 🤔

Reply:
yea youre right